### DISCUSSION TITLE
災情？

### DISCUSSION THREAD

**Discussion by jerry850313** [1101-GS4719-C/Python-Community-GS4719-C#97] [General]
聽說有人python有更新異常，不知道大家有沒有遇到?

**Comment by enoch-lightning** [comment]
不知道你在說什麼，有圖片可以參考嗎？

**Reply by jerry850313** [reply]
![Screenshot_20211019-004530~2.png](https://user-images.githubusercontent.com/56184165/137955517-1419d15e-8d78-4592-9ad3-17b44ca999a8.png)

**Comment by chia-kai-chang** [comment]
所以說不建議安裝python到最新版本，安裝到前兩個版本是比較安全的作法。Anaconda的Python一定不是最新的，這也是為了其他套件的相容性。
